02.general/01.adguard-ad-filters/docs.it.md

```diff
@@ -0,0 +1,204 @@
+---
+titolo: 'Filtri per inserzioni di AdGuard'
+tassonomia:
+    categoria:
+        - documenti
+visibile: vero
+---
+
+*   [Introduzione](#introduzione)
+*   [Filtro di base](#inglese)
+*   [Filtro di protezione da monitoraggio](#privacy)
+*   [Filtro dei social media](#social)
+*   [Filtro per elementi fastidiosi](#fastidi)
+*   [Filtro per l'auto-promozione e lo sblocco di pubblicità](#useful)
+*   [Filtro russo](#russo)
+*   [Filtro tedesco](#tedesco)
+*   [Filtro francese](#francese)
+*   [Filtro giapponese](#giapponese)
+*   [Filtro olandese](#olandese)
+*   [Filtro spagnolo/portoghese](#spagnolo)
+*   [Filtro turco](#turco)
+*   [Filtro sperimentale](#sperimentale)
+*   [Filtro per le inserzioni su mobile](#mobile)
+*   [Filtro Safari](#safari)
+*   [Filtro per i nomi di dominio semplificati](#domains)
+
+<a name="introduction"></a>
+
+## Introduzione
+
+Il filtro è una serie di regole di filtraggio riguardanti il contenuto pubblicitario (banners, pop-ups e simili). AdGuard è dotato di una serie di filtri standard creati da noi. Essi sono costantemente migliorati e integrati, e si spera soddisfino la maggior parte degli utenti. Di tanto in tanto, ogni applicazione di AdGuard controlla se sono disponibili aggiornamenti per tutti i filtri pubblicitari abilitati.
+
+Ci sono diverse opzioni per parlarci dei problemi riguardanti i filtri pubblicitari. Puoi inviare le lamentele attraverso l'applicazione o l'estensione browser, o puoi contattarci sul [forum](https://forum.adguard.com/?langid=1).
+
+I filtri sono ideati per l'utilizzo in tutti i prodotti AdGuard. Se lo desideri, puoi anche usarli in Adblock Plus. Nota però che la sintassi delle regole di filatraggio di AdGuard sono più avanzate rispetto a quelle in ABP. Quindi, una parte delle regole non funzionerà in ABP.
```

Comment:
Are you sure of the right use of the verb in the following sentence? It seems to me the verb is applicable to singular "synthax\sintassi", not the plural "rules\regole". There's the corrected version, just in case:  Nota però che la sintassi delle regole di filatraggio di AdGuard è più avanzata rispetto a quella in ABP. Quindi, una parte delle regole non funzionerà in ABP.